doc/user_guide/_docs/A08-modules.md

```diff
@@ -50,9 +50,11 @@ class MyModule extends Module {
 
 All gates or functionality apart from assign statements in ROHD are implemented using Modules.
 
-#### Inputs, outputs, widths, and getters
+#### Ports, widths, and getters
 
-The default width of an input and output is 1.  You can control the width of ports using the `width` argument of `addInput()` and `addOutput()`.  You may choose to set them to a static number, based on some other variable, or even dynamically based on the width of input parameters.  These functions also return the input/output signal.
+The default width of a port is 1.  You can control the width of ports using the `width` argument of `addInput()` and `addOutput()`.  You may choose to set them to a static number, based on some other variable, or even dynamically based on the width of input parameters.  These functions also return the input/output signal.
+
+There are also similar functions called `addMatchedInput` and `addMatchedOutput` which will create a port with matching widths and types.  This is especially useful for creating `LogicStructure` ports.
```

Comment:
What about Arrays?  Should these all be listed in one place so people know the variety and can itemize them?